core/shared/platform/linux-sgx/sgx_ipfs.c

```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright (C) 2022 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#if WASM_ENABLE_SGX_IPFS
```

Comment:
In most cases, we used style like `#if WASM_ENABLE_SGX_IPFS != 0` or `#if WASM_ENABLE_SGX_IPFS == 0` to control the source code